mystyle.css

```diff
@@ -1,21 +1,115 @@
+@import url(https://fonts.googleapis.com/css?family=Raleway:400,500);
+html,
 body {
-    background-color: rgb(4, 125, 155);
+    background-color: #a9d7d1;
     text-align: center;
     font-family: 'Raleway', Helvetica, sans-serif;
     min-width: 320px;
 }
 
-h1 {
-    color: rgb(41, 36, 59);
-    margin-left: 20px;
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif
+header {
+    font-size: 2em;
+    font-weight: bold;
+    margin: 20px;
+}
+
+#form-outer {
```

Comment:
@myrasika  Kindly change all your `id` references to classes. Classes are highly recommended because they take less time to be parsed and applied across the DOM when the page is loading.